pages/linux/pacman-deptest.md

```diff
@@ -0,0 +1,20 @@
+# pacman --deptest
+
+> Check each dependency specified and return a list of dependencies that are not currently satisfied on the system.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --deptest --help`
```

Comment:
Perhaps this should be the last example? I'd assume that we'd have the most common uses first, and then if they want to find out more then this would be the command to use.